FabulousContacts/Controls/BorderedEntry.fs

```diff
@@ -7,27 +7,42 @@ open Xamarin.Forms
 type BorderedEntry() =
     inherit Entry()
 
-    static let BorderColorProperty = BindableProperty.Create("BorderColor", typeof<Color>, typeof<BorderedEntry>, Color.Default)
+    static let borderColorProperty =
+        BindableProperty.Create("BorderColor",
+                                typeof<Color>,
+                                typeof<BorderedEntry>,
+                                Color.Default)
 
     member this.BorderColor
-        with get () = this.GetValue(BorderColorProperty) :?> Color
-        and set (value) = this.SetValue(BorderColorProperty, value)
+        with get () =
+            this.GetValue(borderColorProperty) :?> Color
+        and set (value) =
+            this.SetValue(borderColorProperty, value)
 
 [<AutoOpen>]
 module DynamicViewsBorderedEntry =
-    let BorderedEntryBorderColorAttributeKey = AttributeKey<_> "BorderedEntry_BorderColor"    
+    let BorderedEntryBorderColorAttributeKey =
+        AttributeKey<_> "BorderedEntry_BorderColor"
 
     type Fabulous.XamarinForms.View with
-        static member BorderedEntry(?borderColor: Color,
-                                    ?placeholder, ?text, ?textChanged, ?keyboard) =
+        static member BorderedEntry(?borderColor: Color, ?placeholder, ?text, ?textChanged, ?keyboard) =
             let attribCount = match borderColor with None -> 0 | Some _ -> 1
             let attribs =
-                ViewBuilders.BuildEntry(attribCount, ?placeholder=placeholder, ?text=text, ?textChanged=textChanged, ?keyboard=keyboard)
+                ViewBuilders.BuildEntry(attribCount,
+                                        ?placeholder = placeholder,
+                                        ?text = text,
+                                        ?textChanged = textChanged,
+                                        ?keyboard = keyboard)
 
-            match borderColor with None -> () | Some v -> attribs.Add(BorderedEntryBorderColorAttributeKey, v)
```

Comment:
This shouldn't be changed.

The reason is that we use `match ... with` because we rely on an optimization from the compiler.

With the (currently missing) `inline` operator, if the parameter `borderColor` is not specified by the user, this instruction will be removed during compilation by the F# compiler. Only leaving useful lines.
If we replace that with `Option.map`, this optimization will not be possible.